apps/music/js/ui/tab_bar.js

```diff
@@ -44,6 +44,7 @@ var TabBar = {
   init: function tab_init() {
     this.option = 'mix';
     this.view.addEventListener('touchend', this);
+    this.view.addEventListener('click', this);
```

Comment:
uh oh. I think the better approach is then to figure out how to emit the touchend event.